### PR TITLE
EIP-2266: change status from draft to last call

### PR DESCRIPTION
**Pull request by HAOYUatHZ**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your Github username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - Trying to change EIP 2266 state from Draft to Last Call
 - EIP 2266 requires approval from one of (jiangshan.yu@monash.edu, runchao.han@monash.edu, chris.haoyul@gmail.com)

**Comment by SebastianElvis** [comment]
I'm @SebastianElvis, and I approve this change. Thanks.

File: EIPS/eip-2266.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 eip: 2266
 title: Atomic Swap-based American Call Option Contract Standard
-author: Runchao Han <runchao.han@monash.edu>, Haoyu Lin <chris.haoyul@gmail.com>, Jiangshan Yu <jiangshan.yu@monash.edu>
+author: Runchao Han <runchao.han@monash.edu>, Haoyu Lin <haoyu@protonmail.com>, Jiangshan Yu <jiangshan.yu@monash.edu>
 discussions-to: https://github.com/ethereum/EIPs/issues/2266
-status: Draft
+status: Last Call
```
Comment (axic) [inline EIPS/eip-2266.md]: This is missing a `review-period-end` field.

Comment (HAOYUatHZ) [reply EIPS/eip-2266.md]: Sorry 
I thought it was to be arranged and filled by EIPs editors.

Do you mean I would need to fill it by myself?

Comment (HAOYUatHZ) [reply EIPS/eip-2266.md]: > This is missing a `review-period-end` field.

Sorry
I thought it was to be arranged and filled by EIPs editors.

Do you mean I would need to fill it by myself?

Comment (axic) [reply EIPS/eip-2266.md]: > Do you mean I would need to fill it by myself?

Yes.

**Comment by HAOYUatHZ** [comment]
> I think it is too early going into last call, given the draft was written up/merged a month ago. There doesn't seem to be a lot of community discussion on this ERC, hence last call seems premature.

Hi Alex @axic 

I think ur point makes sense. It will not be a good a idea to make a PR final, without active discussions. We all hope to mitigate the risk of an EIP not being well designed.

Actually we did open [a thread](https://ethereum-magicians.org/t/eip-xxx-fair-atomic-swap-contract/3549) on ethereum-magicians a couple of months ago, but seems no one replied, :(.

Actually this EIP is inspired by [our paper](https://eprint.iacr.org/2019/896), published at [The First ACM Conference on Advances in Financial Technologies (AFT’19)](http://aft.acm.org/).  AFT19's committee chairs includes: Ittai Abraham (VMware research), Ittay Eyal (Technion, IC3) & Emin Gün Sire (Cornell University).

And our presentation in ETH Zurich is liked by Emin Gün Sire. His tweet on it, https://twitter.com/el33th4xor/status/1186211608596533248, also received attentions from the community.

Therefore, we are confident about our design.

So what will be ur suggestion? Do I need to open another thread on  https://ethresear.ch/ or somewhere else?

**Comment by HAOYUatHZ** [comment]
Hi @axic what can we do?

**Comment by HAOYUatHZ** [comment]
Hi @axic . I see. I have added `review-period-end` field.

**Comment by axic** [comment]
>  I have added review-period-end field.

@HAOYUatHZ the status still needs to be set as "Last Call" for this to take effect. It seems the validator allows the stray `review-period-end` field though.

Other review comment: this draft is missing the "Security Considerations" section.

**Pull request merged**